streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1963,7 +1924,6 @@ public String toString() {
                 + nodeNames(predecessors);
         }
 
-        @SuppressWarnings("unchecked")
```

Comment:
We don't get an actually generic type check below, as this warning tells us... Removing types and going with `<?>` is sufficient below, and allows to to remove this suppression